src/components/CheckboxField/index.tsx

```diff
@@ -70,19 +63,12 @@ const CheckboxField = forwardRef(
       value,
     })
 
-    const error = useMemo(
-      () =>
-        meta.error && meta.touched
-          ? getFirstError({
-              allValues: values,
-              label,
-              meta: meta as FieldState<string | boolean | undefined>,
-              name,
-              value: input.value ?? input.checked,
-            })
-          : undefined,
-      [getFirstError, input.checked, label, meta, name, values, input.value],
-    )
+    const error = getError({
+      label,
+      meta: meta as FieldState<unknown>,
+      name,
+      value: input.value ?? input.checked,
+    })
```

Comment:
Mis `allValues` it can be useful

Reply:
Check `providers/ErrorContext/index.tsx` I get it here